.circleci/config.yml

```diff
@@ -74,10 +100,11 @@ jobs:
             }
           allowed_tools: "Bash,Edit,FileRead,WebSearch,mcp__circleci-mcp-server"
 
-
 workflows:
   ai-cli-setup:
     jobs:
       - install-ai-clis
       - test-claude-mcp:
-          context: agentic
\ No newline at end of file
+          context: agentic
+          requires:
```

Comment:
🔗 **Smart workflow dependency addition!** Adding `requires: - install-ai-clis` ensures the MCP test only runs after CLIs are successfully installed. This prevents test failures due to missing dependencies.